src/test/gcp/cloudscheduler.spec.ts

```diff
@@ -0,0 +1,104 @@
+import * as _ from "lodash";
+import { expect } from "chai";
+import * as nock from "nock";
+import * as api from "../../api";
+
+import { cloudscheduler } from "../../gcp/";
```

Comment:
Why is there a trailing slash after `gcp`? Is that strictly necessary?

Reply:
nope, removing